### DISCUSSION TITLE
Prism Template Pack: VS 2022 support

### DISCUSSION THREAD

**Discussion by glautrou** [PrismLibrary/Prism#2612] [WPF] [answered]
Is there a way to install the "_Prism Template Pack_" for _Visual Studio **2022**_?

I can import snippets manually, but unfortunately not the PRISM "Add new item" templates.
"Prism UserControl (WPF)" is needed for me, I didn't find any hack.

**Comment by dansiegel** [comment, marked as answer]
The move to 64 bit required extension authors to make updates before the extension will be compatible. Right now both @brianlagunas and I are buried with work from those pesky day jobs. But it'll come as soon as Brian has time to update it.

**Reply by BeatSkip** [reply]
I haven't fully verified the functionality. But my workaround seems to be fully working. I simply Extracted the VSIX file from the marketplace as a zipfile, added the following line in the "extension.vsixmanifest" file as an extra InstallationTarget. 

then i repacked the VSIX folder as a zip, renamed it to *.vsix and everything seems to be working. I'd make a pull request, but as there is no repo for the vs marketplace version, that is not an option. 

Update: while adding items sometimes a messagebox pops up with an exception. Although works fine otherwise. So it really is just a temporary workaround to work with 2022. But does the job

```xml
<InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[17.0,18.0)">
      <ProductArchitecture>amd64</ProductArchitecture>
</InstallationTarget>

**Reply by czyt** [reply]
I installed extention successed.prism user control can be created but no auto generated viewmodel,it should be created manually
